.github/workflows/pic-action.yml

```diff
@@ -0,0 +1,46 @@
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+
+jobs:
+  test_pocket_ic_server_action:
+    runs-on: ${{ matrix.os }}
+    name: Test the PocketIC server action (action.yml)
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - version: '5.0.0'
+            expected-version: '5.0.0'
+            os: ubuntu-latest
+          - version: '5.0.0'
+            expected-version: '5.0.0'
+            os: macos-latest
+
+          - version: 'latest'
+            expected-version: '6.0.0'
+            os: ubuntu-latest
+          - version: 'latest'
+            expected-version: '6.0.0'
+            os: macos-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Install PocketIC server and check its version
+        uses: ./
+        with:
+          pocket-ic-server-version: ${{ matrix.version }}
+      - name: Check if the proper PocketIC server version is installed
+        run: |
+          actual_pic_ver="$(${POCKET_IC_BIN} --version)"
```

Comment:
If this is unset or no binary exists this will fail. Is that ok? Will it just evaluate to an err string?

Reply:
> Is that ok?

That's desirable since we're testing the action in this CI job.

> Will it just evaluate to an err string?

I don't think so, the CI job will fail if `${POCKET_IC_BIN}` is not set (which is what we want).